src/chat-lang.pt_BR.js

```diff
@@ -0,0 +1,10 @@
+/* language file */
+const lang = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['lang' is assigned a value but never used.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482470)